packages/web-app-files/src/components/AppBar/SelectedResources/BatchActions.vue

```diff
@@ -127,11 +127,7 @@ export default {
         return false
       }
 
-      if (this.isPublicFilesRoute) {
-        return this.currentFolder.canCreate()
-      }
-
-      return true
+      return this.currentFolder?.canCreate()
```

Comment:
IMO this check is too restrictive. For the `copy` action being available it is sufficient that *any* folder exists in your home where you have write access. Not relevant for the current folder. What's the reasoning here? 🤔

Reply:
Hi @kulmann you are right - in our particular use case files only exist within shares, which inherit the permissions from the root share, so it's effectively the same as testing `canCreate()` on the current effective share.

For the copy action to have a chance of being successful, it would need to have the ability to READ the file in question, and to be able to WRITE to at least one place that is currently mounted.  I added this restriction in our codebase as part of implementing ownclouds `download` permission.  We don't want people being able to `copy` a file to a place where they have elevated permissions and can then download it, effectively working around the permissions model.

Happy to drop this for now and we will keep our customisation, but perhaps this should be testing the 'V' in the permissions instead? `this.permissions.indexOf('V') >= 0`

Reply:
Hmmm. I fully agree that if downloading is forbidden then `canCreate` should evaluate to `false`. Let's sort out the  `canDownload` in your other PR and then use it in this one with a dedicated, new `canCopy` callback on resources.

Reply:
@kulmann i'm just reviewing my PRs in this repo, where did we land with the thinking on this one?